lib/core.js

```diff
@@ -555,6 +555,8 @@ Archiver.prototype.append = function(source, data) {
  * @return {this}
  */
 Archiver.prototype.bulk = function(mappings) {
+  console.log('Archiver.bulk() deprecated since 0.21.0');
```

Comment:
Better do `console.warn` (or `console.error`) it's safer to not pollute the `stdout`.

We also could leverage the new [`process.emitWarning`](https://nodejs.org/api/process.html#process_process_emitwarning_warning_name_ctor)? Only restriction is that it's node 6.0.0+ only.